guides/user/pipeline/searching/index.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Searching for Triggered Pipeline Executions"
+sidebar:
+  nav: guides
+redirect_from: /guides/user/triggers/searching
+---
+
+{% include toc %}
+
+There exists an API that can be used to search for pipeline executions given information about what triggered them. This guide details the best practices to structure your triggers so that their corresponding pipeline executions can be easily queried.
+
+TODO: Link to API here
+
+## Pub/Sub
+
+### Structuring your trigger
+
+Structure your Pub/Sub messages to be unique by adding a random key/value pair for each message. For example, you can add an `id` field for each message with a randomly generated value.
```

Comment:
"When you create the trigger, add an `id` field for which the value will be randomly generated for each message."